src/index.ts

```diff
@@ -111,13 +111,25 @@ export class GoogleToken {
    * @param callback The callback function.
    */
   getToken(): Promise<TokenData>;
-  getToken(callback: GetTokenCallback): void;
-  getToken(callback?: GetTokenCallback): void | Promise<TokenData> {
+  getToken(callback: boolean): Promise<TokenData>;
+  getToken(callback: GetTokenCallback, forceRefresh?: boolean): void;
+  getToken(
+    callback?: GetTokenCallback | boolean,
+    forceRefresh = false
```

Comment:
I really like the options object style as opposed to single-purpose parameters. It's not so bad if everyone is using a TypeScript IDE, but regular old users would have a bit more of a struggle.

Reply:
I agree here. It might seem like overkill, but it is easily extendable making it a bit more future proof.